scripts/setup-ubuntu.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+p=$PWD
```

Comment:
Unquoted is unsafe if any spaces are in path leading to current working directory. Also isn't used anywhere.